package/android/cpp/rnskia-android/RNSkAndroidView.h

```diff
@@ -56,6 +56,7 @@ class RNSkAndroidView : public T, public RNSkBaseAndroidView {
   void surfaceSizeChanged(int width, int height) override {
     std::static_pointer_cast<RNSkOpenGLCanvasProvider>(T::getCanvasProvider())
         ->surfaceSizeChanged(width, height);
+    RNSkView::renderImmediate();
```

Comment:
I believe this would be unnecessary - see comment below about calling `updateTexImage` later.

Reply:
I know this is not great but this will trigger two renders (on available and on resize, the second one invoking the valid call to updateTexImage).